great_expectations/core/batch.py

```diff
@@ -661,6 +661,7 @@ def batch_request_contains_runtime_parameters(
 ) -> bool:
     return (
         batch_request is not None
+        and isinstance(batch_request, (dict, DictDot))
```

Comment:
shouldn't this method's type hints change as well?

Reply:
@donaldheppner I do not think so -- the method still returns `bool`.  Thanks.